### PR TITLE
Tooltip: Improve the accessibility of the composer and the rich text editor

### PR DESCRIPTION
**Pull request by florianduros**
## Checklist

-   [ ] Tests written for new code (and old code if feasible).
-   [ ] New or updated `public`/`exported` symbols have accurate [TSDoc](https://tsdoc.org/) documentation.
-   [ ] Linter and other CI checks pass.
-   [ ] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md)).

Part of https://github.com/element-hq/element-web/issues/26762
Following https://github.com/matrix-org/matrix-react-sdk/pull/12443, `AccessibleButton` includes the new tooltip. We are migrating step by step the usage of the deprecated `AccessibleTooltipButton` to `AccessibleButton`.
Depends of https://github.com/matrix-org/matrix-react-sdk/pull/12458 for `RovingAccessibleTooltipButton` migration.
Depends of https://github.com/element-hq/compound-web/pull/165 to use `<kbd />` for shortcut in tooltips of formatting buttons.

Marking it at `T-Enhancement` because it's in the objective to improve accessibility.

Migrating both composer.

### CODE REVIEW

File: src/components/views/rooms/MessageComposerButtons.tsx [hunk 1]
```diff
@@ -128,10 +127,11 @@ const MessageComposerButtons: React.FC<IProps> = (props: IProps) => {
         <UploadButtonContextProvider roomId={room.roomId} relation={props.relation}>
             {mainButtons}
             {moreButtons.length > 0 && (
-                <AccessibleTooltipButton
+                <AccessibleButton
                     className={moreOptionsClasses}
                     onClick={props.toggleButtonMenu}
                     title={_t("quick_settings|sidebar_settings")}
+                    placement="right"
```
Comment (t3chguy) [inline src/components/views/rooms/MessageComposerButtons.tsx:134]: Isn't this the default?

File: src/components/views/rooms/wysiwyg_composer/components/FormattingButtons.tsx
Comment (t3chguy) [inline src/components/views/rooms/wysiwyg_composer/components/FormattingButtons.tsx:1]: The changes here feel like a visual regression, can @matrix-org/design sign off on losing the keyboard key styling here?

Comment (t3chguy) [reply src/components/views/rooms/wysiwyg_composer/components/FormattingButtons.tsx:1]: Actually my bad, there's no visual regression, only an a11y one. The `kbd` elements are lost and thus also are the semantics

Before:
image

**Review by t3chguy — CHANGES_REQUESTED** [review]
See above comments

File: src/components/views/rooms/wysiwyg_composer/components/FormattingButtons.tsx
Comment (florianduros) [reply src/components/views/rooms/wysiwyg_composer/components/FormattingButtons.tsx:1]: Good catch, will need to update the tooltip compound to be able this use case.

**Pull request merged**